pkg/upgrade/jiva.go

```diff
@@ -50,17 +46,14 @@ type jobInfo struct {
 }
 
 // Jiva Data-plane Upgrade Job instantiator
-func InstantiateJivaUpgrade(openebsNs string) {
+func InstantiateJivaUpgrade(upgradeOpts UpgradeOpts) {
```

Comment:
```suggestion
func InstantiateJivaUpgrade(opts UpgradeOpts) {
```

the variable name and the variable types stutter a bit